munit/shared/src/main/scala-3/munit/internal/MacroCompat.scala

```diff
@@ -10,21 +10,22 @@ object MacroCompat {
     inline implicit def generate: Location = ${ locationImpl() }
   }
 
-  def locationImpl()(using qctx: QuoteContext): Expr[Location] = {
-    import qctx.tasty.{_, given _}
-    val path = rootPosition.sourceFile.jpath.toString
-    val startLine = rootPosition.startLine + 1
+  def locationImpl()(using Quotes): Expr[Location] = {
```

Comment:
Looks like Quotes are not found, I also wonder if this will compile under previous versions? 3.0.0-M1 and 0.27.0-RC1?

I think we can drop 0.27.0-RC1, but we might need to have some campatibility utility classes after all.

Reply:
I was stupid and I didn't consider M1. Should be fixed in the latest commit

Reply:
I don't think we should drop 0.27-RC1 in this PR though, maybe we can discuss the consequences in a different issues